packages/components/src/ErrorPage/ErrorPage.tsx

```diff
@@ -66,7 +75,7 @@ export function ErrorPage({
             </Paragraph>
           </>
         }
-        illustration={<BrandMomentError isAnimated loop />}
```

Comment:
I think we can make a call as to whether or not `ErrorPage` needs to animate at all. Given their relatively low exposure, it seems unnecessary to provide an API for this.

Reply:
It's actually animated by default 😅 But the animation doesn't show in Chromatic, so I wanted a way to stop it.

Reply:
But what I'm saying is set this to `isAnimated={false}`, since I don't think we need an option to animate it in the first place.